.ci.yaml

```diff
@@ -361,22 +361,18 @@ targets:
   # This is a benchmark that does not require an attached device. However, we
   # are intentionally running it in the devicelab to ensure that it does not
   # run on a VM in order to avoid noisy results from the benchmark.
-  - name: Linux analyzer_benchmark
+  - name: Mac analyzer_benchmark
```

Comment:
Not sure I'm remembering correctly, but I think that the `tags` section doesn't actually affect which machine pool is selected. I think the way this is written the task will go on any available intel or arm64 mac. To always select an arm64 mac, this should be `Mac_arm64` instead of `Mac`.

Reply:
done